pkgs/applications/virtualization/lima/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, qemu
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "lima";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "AkihiroSuda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-vwAxVBy2SqghxJGscSZ1o1B8EMvQh1fz5CS1YG7Rq2g=";
+  };
+
+  vendorSha256 = "sha256-xM9LLh5c5QBrcIptdqiNNp1nU9GcdQvwrCnnyuXWYfE=";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  buildPhase = ''
+    runHook preBuild
+    make "VERSION=v${version}" binaries
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r _output/* $out
+    wrapProgram $out/bin/limactl \
+      --prefix PATH : ${lib.makeBinPath [ qemu ]}
+    installShellCompletion --cmd limactl \
+      --bash <($out/bin/limactl completion bash)
+    runHook postInstall
+  '';
+
+  installPhaseCheck = ''
+    $out/bin/limactl prune 2>&1
```

Should really matter as we set $HOME during the build but using a non destructive test is probably safer.

```suggestion
    $out/bin/limactl validate $out/share/doc/lima/examples/default.yaml
```

---

I try build without USER and didn't see the error like yours.

---

btw, i used that check because that's what they use in their homebrew package

---

Same error with `ofborg` https://github.com/NixOS/nixpkgs/pull/128052/checks?check_run_id=2913381618

---

You're right, just now I tested on nixos and not macos. Building on macos gives the same error